<p>Traceback created on "2016-03-22 12:11:17"
<table border=TRUE>
<tr>
<td>Level</td><td>Condition</td><td>Method</td><td>Sourcefile</td><td>Sourceline</td>
</tr>
<tr>
<td>1</td><td>Implicit Euler integration failed! Could not find a solution of requested accuracy 0.001 within 100000 integration steps.</td><td>void euler_im(const std::vector<double>&, unsigned int, abstractObject*, double, unsigned int, std::vector<double>&)</td><td>/home/tobias/Anwendungen/echse/echse_generic/core/echse_coreFunct_solveODE.cpp</td><td>563</td>
</tr>
<tr>
<td>2</td><td>Simulation failed for time step of length 3600.</td><td>virtual void object_test_soilwat::simulate(unsigned int)</td><td>/home/tobias/Anwendungen/echse/echse_engines/generated/test_soilwat/AUTOechse_userClass_test_soilwat.h</td><td>106</td>
</tr>
<tr>
<td>3</td><td>Simulation failed for object 'test1' in time step 1 of 4344 starting at 2001-01-01 00:00:00.</td><td>int main(int, const char**)</td><td>/home/tobias/Anwendungen/echse/echse_generic/core/echse_coreFunct_main.cpp</td><td>508</td>
</tr>
<tr>
<td>4</td><td>1 exceptions registered in time step 1 of 4344.</td><td>int main(int, const char**)</td><td>/home/tobias/Anwendungen/echse/echse_generic/core/echse_coreFunct_main.cpp</td><td>545</td>
</tr>
<tr>
<td>5</td><td>Stopped due to exception. See traceback in 'output_solver_compare/euler_imp_own/test_soilwat.err.html'.</td><td>int main(int, const char**)</td><td>/home/tobias/Anwendungen/echse/echse_generic/core/echse_coreFunct_main.cpp</td><td>605</td>
</tr>
</table>
